orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/cluster/FindImageFromClusterTask.groovy

```diff
@@ -160,7 +160,7 @@ class FindImageFromClusterTask extends AbstractCloudProviderAwareTask implements
         return [(location): summaries]
       } catch (RetrofitError e) {
         if (e.response?.status == 404) {
-          final Map reason
+          Map reason
```

Comment:
I guess Groovy cares about the `final` keyword now. This refused to compile until I changed it.